Marlin/src/module/temperature.cpp

```diff
@@ -944,7 +944,7 @@ void Temperature::manage_heater() {
     if (temp_hotend[1].current < MAX(HEATER_1_MINTEMP, HEATER_1_MAX6675_TMIN + .01)) min_temp_error(1);
   #endif
 
-  #if WATCH_HOTENDS || WATCH_BED || DISABLED(PIDTEMPBED) || HAS_AUTO_FAN || HEATER_IDLE_HANDLER || WATCH_CHAMBER
+  #if WATCH_HOTENDS || WATCH_BED || DISABLED(PIDTEMPBED) || HAS_AUTO_FAN || HEATER_IDLE_HANDLER || WATCH_CHAMBER || ENABLED(THERMAL_PROTECTION_BED) || ENABLED(THERMAL_PROTECTION_CHAMBER)
```

Comment:
At this point it's pretty much always needed. :shrug:

Reply:
after setting hotend temperature about 240 degrees and resetting the board I've got thermal protection error

Reply:
Did the thermal protection error say either bed or chamber triggered it?

Reply:
bed and chamber protection works well.
now I have a problem with Hotend thermal protection.

Reply:
That is a separate issue then. I assume your hotend maxtemp is way higher than the 240 degrees that triggered the protection error, correct? I guess a similar grace period for the hotends might be useful as well.

Reply:
I think the problem is very similiar with bed and chamber, because in some days ago all worked.
Yes, my Hotend max temp is 270 degrees.

Reply:
so, can You add something to fix this problem?

Reply:
Please test the latest commit on this MR, there's a grace period on the hotend measurement as well now.

Reply:
thank you very much. it works.